tests/cpgrid/inactiveCell_lgr_test.cpp

```diff
@@ -294,6 +294,75 @@ void testInactiveCellsLgrs(const std::string& deckString,
             BOOST_CHECK_EQUAL(element.level(), l);
         }
     }
+
+    const auto& global_id_set_ptr = grid.global_id_set_ptr_;
+
+    std::set<int> allIds_set;
+    std::vector<int> allIds_vec;
+    allIds_vec.reserve(data.back()->size(0) + data.back()->size(3));
+    const auto& leaf_view = grid.leafGridView();
+    for (const auto& element: elements(leaf_view)){
+        const auto& localId = data.back()->local_id_set_->id(element);
+        const auto& globalId = data.back()->global_id_set_->id(element);
```

Comment:
Maybe use DUNE's grid methods to access the id sets:
```suggestion
        const auto& localId = grid.localIdSet().id(element);
        const auto& globalId = grid.globalIdSet().id(element);
```

Reply:
Good point. globalIdSet() was not defined for CpGridData. In the coming commit, it's defined and used!

Reply:
For CpGrid, both localIdSet() and globalIdSet() existed already. I misunderstood your comment. In any case, now there is also a method globalIdSet() for CpGridData

Reply:
In any case, for CpGrid, localIdSet() and globalIdSEt() both return the same (*global_id_set_prt_). Therefore, for testing purposes, I call the methods for each level grid and the leaf grid view (and not the entire grid)